examples/game/breakout.rs

```diff
@@ -63,7 +75,7 @@ fn setup(
             ..Default::default()
         })
         .with(Ball {
-            velocity: 400.0 * Vec3::new(0.5, -0.5, 0.0).normalize(),
+            velocity: 400.0 * TIME_STEP * Vec3::new(0.5, -0.5, 0.0).normalize(),
```

Comment:
I don't think we should include TIME_STEP in the velocity itself (as that isn't the definition of velocity). We should use TIME_STEP when computing translation.

Reply:
Same for "speed"